src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,182 @@
+                + "INNER JOIN drivers ON cars_drivers.driver_id = drivers.id "
+                + "WHERE cars.is_deleted = FALSE;";
+        List<Car> cars = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                cars.add(createCar(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't add car drivers.", e);
+        }
+        return cars;
+    }
+
+    @Override
+    public Car update(Car car) {
+        String query = "UPDATE cars SET manufacturer_id = ?, cars.model = ? WHERE cars.id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, car.getManufacturer().getId());
+            statement.setString(2, car.getModel());
+            statement.setLong(3,car.getId());
+            statement.executeUpdate();
+            deleteRelations(car.getId());
+            insertCarDrivers(car);
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't add car drivers.", e);
+        }
+        return car;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String query = "UPDATE cars SET is_deleted = TRUE WHERE id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            return statement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't add car drivers.", e);
+        }
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        String query = "SELECT * FROM cars "
+                + "INNER JOIN manufacturers ON manufacturers.id = cars.manufacturer_id "
+                + "INNER JOIN cars_drivers ON cars_drivers.car_id = cars.id "
+                + "INNER JOIN drivers ON cars_drivers.driver_id = drivers.id "
+                + "WHERE driver_id = ?;";
+        List<Car> cars = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, driverId);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                cars.add(createCar(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't add car drivers.", e);
+        }
+        return cars;
+    }
+
+    private Car createCar(ResultSet resultSet) throws SQLException {
```

Comment:
in this method only get car, don't put drivers here, create a separate method, that makes a request to get drivers by this car